recipes/genoboo/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set name = 'genoboo' %}
+{% set version = '0.4.0' %}
+{% set sha256 = '0acce8435a6996f525f92daf22d6f8952703659dc8dd528b5d071b5c72a9bfe7' %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/gogepp/genoboo/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+requirements:
+  host:
+    - jq
+    - nodejs >=15, <16
+    - python >=3
+  build:
+    - {{ compiler('cxx') }}
+    - make
+  run:
+    - blast
+    - nodejs >=15, <16
+    - mongodb >=5.0
+
+build:
+  number: 0
+
+test:
+  commands:
+    - genoboo -v
+
+about:
+  home: https://github.com/gogepp/genoboo
+  license: AGPL-3.0
```

Comment:
Not sure how to manage the meteor license  https://github.com/meteor/meteor/blob/devel/LICENSE

Reply:
I see meteor as an MIT framework/library, used by genoboo which itself is AGPL-3.0, isn't that possible?
Of course, if/when we get a meteor package it will be clear the license of each part